OSDOCS-18782 adding ESO docs to 4.19 for GA (#109778)
openshift/openshift-docs: modules/eso-cert-manager-config.adoc

| Default
| Validation

| `mode`
| _string_

> | `mode` specifies whether to use cert-manager for certificate management instead of the built-in `cert-controller` which can be indicated by setting either `Enabled` or `Disabled`. If set to `Enabled`, uses `cert-manager` for obtaining the certificates for the webhook server and other components. If set to `Disabled`, uses the `cert-controller` for obtaining the certificates for the webhook server. `Disabled` is the default behavior.

Reviewer: ```suggestion
| `mode` specifies whether to use the cert-manager Operator for certificate management instead of the built-in `cert-controller` controller. If set to `Enabled`, the Operator uses cert-manager for obtaining the certificates for the webhook server and other components. If set to `Disabled`, the Operator uses the `cert-controller` for obtaining the certificates for the webhook server. `Disabled` is the default behavior.
```
confusing read, suggest simplifying the first sentence
also needs an actor for "uses"

Author: Done

| false
| enum: [true false]

Required

| `injectAnnotations`
| _string_

> | `injectAnnotations` adds the `cert-manager.io/inject-ca-from` annotation to the webhooks and custom resource definitions (CRDs) to automatically configure the webhook with the `cert-manager` Operator certificate authority (CA). This requires CA Injector to be enabled in `cert-manager` Operator. Set this field to `true` or `false`. When set, this field cannot be changed.

Reviewer: ```suggestion
| `injectAnnotations` adds the `cert-manager.io/inject-ca-from` annotation to the webhooks and custom resource definitions (CRDs) to automatically configure the webhook with the `cert-manager` Operator certificate authority (CA). This requires CA Injector to be enabled in  the cert-manager Operator. Set this field to `true` or `false`. When set, this field cannot be changed.
```

Author: Done
